.circleci/config.yml

```diff
@@ -0,0 +1,111 @@
+version: 2
+workflows:
+  version: 2
+  test_upload:
```

Comment:
It might be nice to have two workflows, one for PRs and one for master, just so the workflow names in CI reflect what actually happened. I don't know off hand how to do that though.

Reply:
Yeah I can't find per-workflow filtering, only per-job filtering.